lib/client/states/state.py

```diff
@@ -11,7 +11,7 @@ def leave(self):
         #buh bye - cleanup stuff
         pass
 
-    def render(self):
+    def render(self, screen):
```

Comment:
Technically part of how i was going to do viewport centering originally, but it's generally cleaner for a render func to take its graphics context as an arg rather than reaching out for it.